components/formats-api/src/loci/formats/FormatTools.java

```diff
@@ -1437,7 +1438,50 @@ public static Time getTime(Double value, String unit) {
     }
     return new Time(value, UNITS.SECOND);
   }
-  
+
+
+  /**
+   * Formats the input value for the position into a length of the given unit.
+   *
+   * @param value  the value of the position
+   * @param unit   the unit of the position
+   *
+   * @return       the position formatted as a {@link Length}
```

Comment:
:+1: I will update the Javadocs in a final commit once we have all signed off on the expected behavior.